lib/kino_explorer/data_frame_cell.ex

```diff
@@ -0,0 +1,332 @@
+defmodule KinoExplorer.DataFrameCell do
+  @moduledoc false
+
+  use Kino.JS, assets_path: "lib/assets/data_frame_cell"
+  use Kino.JS.Live
+  use Kino.SmartCell, name: "Data Transform"
```

Comment:
We usually don't capitalize:

```suggestion
  use Kino.SmartCell, name: "Data transform"
```

Let's also mirro this in the module/file names.

@josevalim should we be more specific, like `Dataframe transform`?